lib/galaxy/managers/folders.py

```diff
@@ -563,9 +559,4 @@ def __decode_id(self, trans, encoded_id, object_name=None):
         :param  object_name:      Name of the object the id belongs to. (optional)
```

Comment:
You can probably remove this helper and just replace its use with `trans.security.decode_id()` .